registry.k8s.io/images/k8s-staging-scl-image-builder/images.yaml

```diff
@@ -32,6 +32,7 @@
     "sha256:36b2386ed62631235721dc8ef42a818a624f1ec2d81570be2edc621454034ccc": ["v0.1.40"]
     "sha256:7763ca632d42c8bb5e7f1a9e4dab4fb1e9e352a44aac44984502f9ecafcdff46": ["v0.1.41"]
     "sha256:bd76789c56ffbc44730699fadaea67920e57e73187a18e4967a5e686284a5eff": ["v0.1.42"]
+    "sha256:5a4673ac6a7b8d8b6327e31513b1e2fb37f0d830361e15a136ca5bc9384d3ebe": ["v0.1.43"]
```

Comment:
/hold cancel

`5a4673ac6a7b` matches the SHA in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-scl-image-builder/global/cluster-node-image-builder-amd64@sha256:5a4673ac6a7b8d8b6327e31513b1e2fb37f0d830361e15a136ca5bc9384d3ebe/details?inv=1&invt=AbxfnA) 👍🏻